src/languages/php.js

```diff
@@ -102,11 +103,50 @@ export default function(hljs) {
     // <https://www.php.net/manual/en/reserved.classes.php>
     'Directory __PHP_Incomplete_Class parent php_user_filter self static stdClass'
   };
+  const ATTRIBUTES = {
+    className: 'meta',
+    begin: '^\\s*#\\[',
+    end: '\\]',
+    contains: [
+      {
+        begin: /\[/,
+        end: /\]/,
+        contains: [
```

Comment:
```suggestion
        end: /\]/,
        keywords: { 
          literals: LITERALS,
          keyword: "new",
        },
        contains: [
```

Keywords should work just fine here.

Reply:
Will see.